src/test/java/bio/terra/service/auth/iam/sam/SamIamTest.java

```diff
@@ -67,10 +67,12 @@
 import org.junit.jupiter.api.extension.ExtendWith;
 import org.mockito.Mock;
 import org.mockito.junit.jupiter.MockitoExtension;
+import org.springframework.test.context.ActiveProfiles;
 
 @ExtendWith(MockitoExtension.class)
+@ActiveProfiles("unittest")
 @Tag("bio.terra.common.category.Unit")
```

Comment:
Can this also use the `Unit.TAG`?

Reply:
Yes, thanks for noticing this.. I think this one I may have used intellij or copilot to port over and didn't look at the output too closely.